### PR TITLE
Bugfix/Add-tests-to-ubs-admin-employee-component

### PR DESCRIPTION
**Pull request by NadiiaStoiko**
1. Remove code smell
2. Transfer enums to another file
3. Add tests to ubs-admin-employee-component

### CODE REVIEW

File: src/app/ubs/ubs-admin/components/ubs-admin-employee/ubs-admin-employee-edit-form/ubs-admin-employee-edit-form.component.ts [hunk 1]
```diff
@@ -147,10 +147,12 @@ export class UbsAdminEmployeeEditFormComponent implements OnInit, OnDestroy {
...
       if (this.editMode) {
         this.tariffsFromEditForm = this.tariffsFromEditForm.map((editItem) => editItem.id);
-        this.filteredTariffs = this.tariffs.map((tariffItem) => ({
-          ...tariffItem,
-          selected: this.tariffsFromEditForm.includes(tariffItem.id) ? (tariffItem.selected = true) : tariffItem.selected
-        }));
+        this.filteredTariffs = this.tariffs.map((tariffItem) => {
+          return {
+            ...tariffItem,
+            selected: this.tariffsFromEditForm.includes(tariffItem.id) ? true : false
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-employee/ubs-admin-employee-edit-form/ubs-admin-employee-edit-form.component.ts]: selected: this.tariffsFromEditForm.includes(tariffItem.id)

File: src/app/ubs/ubs-admin/components/ubs-admin-employee/ubs-admin-employee.component.ts [hunk 2]
```diff
@@ -684,4 +666,14 @@ export class UbsAdminEmployeeComponent implements OnInit {
       panelClass: 'admin-cabinet-dialog-container'
     });
   }
+
+  public getExistingCard(filterData) {
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-employee/ubs-admin-employee.component.ts]: this parameter is not used in the method

File: src/app/ubs/ubs-admin/components/ubs-admin-employee/ubs-admin-employee.component.ts [hunk 3]
```diff
@@ -684,4 +666,14 @@ export class UbsAdminEmployeeComponent implements OnInit {
...
+    this.cardsUk.length = 0;
+    this.cardsEn.length = 0;
+  }
+
+  ngOnDestroy(): void {
+    this.destroy.next();
+    this.destroy.complete();
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-admin/components/ubs-admin-employee/ubs-admin-employee.component.ts]: Use takeUntil for unsubscribing

**Review by atsa21 — APPROVED** [review]

**Review by AlexW8719 — APPROVED** [review]

File: src/app/ubs/ubs-admin/components/ubs-admin-employee/ubs-admin-employee-edit-form/ubs-admin-employee-edit-form.component.ts (hunk 1 above)
Comment (NadiiaStoiko) [reply src/app/ubs/ubs-admin/components/ubs-admin-employee/ubs-admin-employee-edit-form/ubs-admin-employee-edit-form.component.ts]: changed according to comment

File: src/app/ubs/ubs-admin/components/ubs-admin-employee/ubs-admin-employee.component.ts (hunk 2 above)
Comment (NadiiaStoiko) [reply src/app/ubs/ubs-admin/components/ubs-admin-employee/ubs-admin-employee.component.ts]: method was removed

File: src/app/ubs/ubs-admin/components/ubs-admin-employee/ubs-admin-employee.component.ts (hunk 3 above)
Comment (NadiiaStoiko) [reply src/app/ubs/ubs-admin/components/ubs-admin-employee/ubs-admin-employee.component.ts]: ngOnDestroy() removed

**Review by ruslan-teh — APPROVED** [review]

**Review by KKatiaa — APPROVED** [review]

**Review by katia9261 — APPROVED** [review]

**Review by hnativlyubomyr — APPROVED** [review]

**Pull request merged**